Formula/c/corrosion.rb

```diff
@@ -1,8 +1,8 @@
 class Corrosion < Formula
   desc "Easy Rust and C/C++ Integration"
   homepage "https://github.com/corrosion-rs/corrosion"
-  url "https://github.com/corrosion-rs/corrosion/archive/refs/tags/v0.5.2.tar.gz"
-  sha256 "6bc02411e29183a896aa60c58db6819ec6cf57c08997481d0b0da9029356b529"
+  url "https://github.com/corrosion-rs/corrosion/archive/refs/tags/v0.6.tar.gz"
+  sha256 "9b4c0daf8cc5b1837d8365e82266f2de01c1bcab9a97a9eff1c929b96a158786"
```

Comment:
Probably want to use [v0.6.0](https://github.com/corrosion-rs/corrosion/tree/v0.6.0) rather than [v0.6](https://github.com/corrosion-rs/corrosion/releases/tag/v0.6) (same commit but former is marked as GitHub release and has release notes)